lua/entities/gmod_wire_expression2/core/custom/cfc_vector2_NumberOperands.lua

```diff
@@ -0,0 +1,396 @@
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		1/tan(rv1 * deg2rad),
+		1/tan(rv2 * deg2rad)
+	}
+end)
+
+registerFunction("rcos", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		cos(rv1),
+		cos(rv2)
+	}
+end)
+
+registerFunction("rsec", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		1/cos(rv1),
+		1/cos(rv2)
+	}
+end)
+
+registerFunction("rsin", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		sin(rv1),
+		sin(rv2)
+	}
+end)
+
+registerFunction("rcsc", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		1/sin(rv1),
+		1/sin(rv2)
+	}
+end)
+
+registerFunction("rtan", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		tan(rv1),
+		tan(rv2)
+	}
+end)
+
+registerFunction("rcot", "xv2", "xv2" function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		1/tan(rv1),
+		1/tan(rv2)
+	}
+end)
+
+registerFunction("acosrvec2", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		acos(rv1),
+		acos(rv2)
+	}
+end)
+
+registerFunction("acosr", "xv2", "n", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	local hypo = sqrt(rv1*rv1+rv2*rv2)
+	return acos(rv1/hypo)
+end)
+
+registerFunction("asinrvec2", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		asin(rv1),
+		asin(rv2)
+	}
+end)
+
+registerFunction("asinr", "xv2", "n", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	local hypo = sqrt(rv1*rv1+rv2*rv2)
+	return acos(rv2/hypo)
+end)
+
+registerFunction("atanrvec2", "xv2", "xv2", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return {
+		atan(rv1),
+		atan(rv2)
+	}
+end)
+
+registerFunction("atanr", "xv2", "n", function(self, args)
+	local op1, op2 = args[2][1], args[2][2]
+	local rv1, rv2 = op1[1](self, op1), op2[1](self, op2)
+	return atan(rv2/rv1)
+end)
+
+--[[************************************************************************]]--
+
+__e2setcost(15) -- approximation
+
+
+e2function vector2 toStringV2(vector2 vector)
+	return {
+    	tostring(vector[1]),
+    	tostring(vector[2])
+  	}
+end
+
+e2function vector2 vector2:toStringV2()
+	return {
```

Comment:
What does this do?

Reply:
Both of these two output a vector2 with a string in them with whatever the tostring is, but now that i see about it, these are both unnecessary and probably wont work

Reply:
Deleting them